quiz_answers.py

```diff
@@ -365,6 +432,17 @@
     .date
 )
 questions[7].expected = latest_moderate_or_severe_frailty_date
+questions[7]._hint = (
+    (
+        "You'll need to either:\n\n"
+        " - combine the moderate and severe frailty codelists, which can be done like this: "
+        "moderate_frailty_codes + severe_frailty_codes\n"
+        " - or make use of the 'or' operator '|' to find snomedct_codes in "
+        "moderate_frailty_codes OR in severe_frailty_codes."
```

Comment:
hmmm, so both `....is_in(moderate + severe)` and `....is_in(moderate) | ....is_in(severe)` are valid? what about `....is_in(moderate | severe)`?

Reply:
anyway, I came here to say I'd like to see the or operator used in the example, but then when I try it's basically not possible to do that in a good way, so... 🤷 

```suggestion
        " - combine the moderate and severe frailty codelists, which can be done like this: "
        "moderate_frailty_codes + severe_frailty_codes\n"
        " - or make use of the 'or' operator '|' to find snomedct_codes in "
        "is_in(moderate_frailty_codes) | is_in(severe_frailty_codes)"
```

Reply:
I guess I was trying to not spoon feed the answer, so tell people about the operators, then tell them what to do in pseudo code and let them figure out the rest.

Reply:
yep fine